package.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "ffmpeg-static",
+  "name": "ffmpeg-ffprobe-static",
```

Comment:
I'd like to keep `ffmpeg-static` as a name, because a) it is already a known name, and b) the name "ffmpeg" is arguably a lot more well-known than "ffprobe".